<launch>
    <arg name="use_thresholding" default="false"/>

    <node pkg="object_follower" type="object_follower_node" name="object_follower" output="screen">
        <remap from="input_image" to="/realsense/color/image_raw" />
    </node>

    <group if="$(arg use_thresholding)">
        <node pkg="object_follower" type="color_thresholding_node" name="color_thresholding_node" output="screen">
            <remap from="threshold_image_input" to="/realsense/color/image_raw" />
        </node>
    </group>
</launch>